selfdrive/test/test_cpu_usage.py

```diff
@@ -101,6 +101,8 @@ def test_cpu_usage():
 
   passed = False
   try:
-    passed = test_cpu_usage()
+    # TODO: remove loop
+    for _ in range(200):
+      passed = test_cpu_usage()
```

Comment:
this doesn't do what you want. you want to exit immediately on failure. the exit status will only reflect the last test, and you'll have to scroll through 200 tests to verify they all passed